SPCTR-3345 Fix: Table of Contents - Resolved an issue where headings containing Latin characters were not functioning correctly. (#3486)
brainstormforce/ultimate-addons-for-gutenberg: README.md

* Fix: Forms - Resolved an issue wherein the Forms block would not work inside Containers in Astra's Custom Layout.
* Fix: Global - Removed the Block Editor Spacing setting for FSE themes.
* Fix: Image - Resolved an issue in which masked images would get cut-off when the zoom-in hover effect was used.

> * Fix: Table of Contents - Resolved an issue where headings containing some of the latin characters were not functioning correctly.

Reviewer: ```suggestion
* Fix: Table Of Contents - Resolved an issue wherein headings containing some uppercase latin characters would not function as intended.
```

Author: Done
